rspace/src/main/scala/coop/rchain/rspace/InMemoryStore.scala

```diff
@@ -261,7 +261,7 @@ object InMemoryStore {
     implicit val codecC: Codec[C] = sc.toCodec
     implicit val codecP: Codec[P] = sp.toCodec
     implicit val codecA: Codec[A] = sa.toCodec
-    initialize(trieStore, branch)
+
```

Comment:
Why is this not needed anymore?

Reply:
With latest update InMemoryStore implementation is closer to LMDBStore, which doesn't need this call as well.